SFLoggingKit/Classes/SFLogWriter.swift

```diff
@@ -0,0 +1,385 @@
+//
+//  SFLogWriter.swift
+//  SFLoggingKit
+//
+//  Created by Aleksandar Sergeev Petrov on 7.07.20.
+//
+
+import Foundation
+import os.log
+
+// NOTE: Writers don't append new line automatically.
+// If proper add it in log message string
+
+/// The `SFLogWriter` protocol defines a single API for writing a log message. The message can be written in any way the conforming object sees fit.
+/// For example, it could write to the console, write to a file, remote log to a third party service, etc.
+public protocol SFLogWriter {
+
+    func log(_ message: String, logLevel: SFLogLevel)
+}
+
+// MARK: - Xcode Console
+
+/// The SFConsoleLogWriter class runs all modifiers in the order they were created and prints the resulting message  to the Xcode console.
+open class SFConsoleLogWriter: SFLogWriter {
```

Comment:
[optional]

Could we create different files for the classes in this one?
I know they are connected but the file is 400+ lines from what I see.